### DISCUSSION TITLE
Populating NetBoxTable with Dict instead of QuerySet

### DISCUSSION THREAD

**Discussion by emersonfelipesp** [netbox-community/netbox#11946] [Q&A] [answered]
I'm testing using NetBoxTable and it works fine with querysets, but I am not being successful using it with dict data, although the default `tables.Table` class which NetBoxTable inherits accepts it. 

Is it a expected behavior?

Thank you already!

**Comment by candlerb** [comment, marked as answer]
Can you show your code, and what error it gives?

Are there particular features of NetBoxTable you need to use? If you don't want to use querysets, then could you use django_tables2.Table [directly](https://django-tables2.readthedocs.io/en/latest/pages/table-data.html#list-of-dicts)?

**Reply by emersonfelipesp** [reply]
Using **django_tables2.Table** directy worked. I just had to add some divs around the table to show it like Netbox

### View
````
class ProxmoxFullUpdate(PermissionRequiredMixin, View):
    virtualmachines_table = VMUpdateResult(vm_data)
    nodes_table = NodeUpdateResult(node_data)

    def get(self, request):
        """Get request."""
        return render(
            request,
            "netbox_proxbox/proxmox_vm_full_update.html",
            {
                "nodes_table": self.nodes_table,
                "virtualmachines_table": self.virtualmachines_table
            },
        )
````

### Template
```
{% extends 'base/layout.html' %}
{% load render_table from django_tables2 %}

{% block content %}
    <h2>Virtual Machines and Containers</h2>
    <div class="card">
        <div class="card-body" id="object-list">
            <div class="table-responsive">
                {% render_table virtualmachines_table %}
            </div>
        </div>
    </div>

    <h2>Nodes</h2>
    <div class="card">
        <div class="card-body" id="object-list">
            <div class="table-responsive">
                {% render_table nodes_table %}
            </div>
        </div>
    </div>
{% endblock %}
```

### Result

![image](https://user-images.githubusercontent.com/24397251/224340537-18278f07-553e-411f-b53e-5800925996f6.png)
![image](https://user-images.githubusercontent.com/24397251/224340618-8956dc1a-aed6-48af-b846-16a132b3d60c.png)


Thank you!
